addons/website/static/tests/tours/link_tools.js

```diff
@@ -328,9 +351,16 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
     },
     */
     // 11. Pick a URL with auto-complete
+    {
+        content: "Wait the sidebar is openend",
+        trigger: `we-title:contains(Inline text)`,
+        extra_trigger: `input#o_link_dialog_url_input`,
+        isCheck: true,
+    },
```

Comment:
Also add a TODO here, this is imo a bug in the hoot framework, I don't see why we need a step to wait for `input#o_link_dialog_url_input` as it's already the trigger of next step

Reply:
No longer needed